### PR TITLE
entity created

### PR DESCRIPTION
**Pull request by taos-thr**

### CODE REVIEW

File: api/src/main/java/com/api/entities/EvaluationForm.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.api.entities;
+
+import lombok.Data;
+
+import java.util.List;
+@Data
+public class EvaluationForm {
+    int id;
+    List<Evaluation>evaluations;
```
Comment (tekcay) [inline api/src/main/java/com/api/entities/EvaluationForm.java]: Requires a space

File: api/src/main/java/com/api/entities/Responsable.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.api.entities;
+
+import lombok.Data;
+
+@Data
+public class Responsable {
+    int id;
+    String name;
+    String mail;
+    public Responsable(){
```
Comment (tekcay) [inline api/src/main/java/com/api/entities/Responsable.java:11]: Skip line between attributes and constructor for readability

**Review by tekcay — CHANGES_REQUESTED** [review]
All attributes must be private.

File: api/src/main/java/com/api/entities/EvaluationForm.java (hunk 1 above)
Comment (taos-thr) [reply api/src/main/java/com/api/entities/EvaluationForm.java]: done

File: api/src/main/java/com/api/entities/Responsable.java (hunk 2 above)
Comment (taos-thr) [reply api/src/main/java/com/api/entities/Responsable.java:11]: done

**Review by tekcay — APPROVED** [review]

**Pull request merged**